mailbox/opensearch/src/test/java/org/apache/james/mailbox/opensearch/ElasticSearchIntegrationTest.java

```diff
@@ -317,12 +315,18 @@ void addressMatchesShouldBeExact() throws Exception {
                     .build()),
             session).getId();
 
-        awaitForElasticSearch(QueryBuilders.matchAllQuery(), 15);
+        awaitForElasticSearch(QueryBuilders.matchAll().build()._toQuery(), 15);
 
         assertThat(Flux.from(messageManager.search(SearchQuery.of(SearchQuery.address(SearchQuery.AddressType.To, "bob@other.tld")), session)).toStream())
             .containsOnly(messageId2.getUid());
     }
 
+    @Disabled("JAMES-3771 Waiting for a fix on opensearch client to be merged: https://github.com/opensearch-project/opensearch-java/pull/169")
+    @Test
+    public void sortOnCcShouldWork() throws Exception {
```

Comment:
I want to wait a bit for this as well. Normally a new client with the fix should be released very soon

Reply:
Annnnd... it's out :)